internal/provider/resource_consul_cluster.go

```diff
@@ -421,18 +411,16 @@ func setConsulClusterResourceData(d *schema.ResourceData, cluster *consulmodels.
 		return err
 	}
 
-	if err := d.Set("scale", cluster.Config.CapacityConfig.NumServers); err != nil {
+	if err := d.Set("scale", cluster.Config.CapacityConfig.Scale); err != nil {
 		return err
 	}
 
-	// TODO: Update this logic when tier becomes a first class value on the cluster
-	for t, numServers := range consulTierToNumServers {
-		if numServers == cluster.Config.CapacityConfig.NumServers {
-			if err := d.Set("tier", t); err != nil {
-				return err
-			}
-			break
-		}
+	if err := d.Set("tier", strings.ToLower(string(cluster.Config.Tier))); err != nil {
```

Comment:
Nit: Since you're already using `DiffSuppressFunc`, I don't think you need to normalize this with `ToLower` (same with size)

Reply:
Done!